### ISSUE TITLE
🇨🇳 Support Chinese

### ISSUE THREAD

**Issue by kiner-tang** [teaxyz/white-paper#323]
With a large number of contributors from China active in the open source community, Tea, as a decentralized platform for ranking open source projects and providing maintenance rewards for open source contributors, has made it necessary to add a Chinese-specific translation to the white paper. If this project accepts PR, I would love to do this work.

**Comment by yanboishere** [comment]
I totally agree bro. I also want to ask if we need to work together to set up a related Chinese community?
